Miscellaneous/docker/full_stack_example/docker-compose-windows.yml

```diff
@@ -0,0 +1,174 @@
+---
+version: "2.1"
+services:
```

Comment:
BUT if we need multiple yml files we can also nest them so that `docker-compose -f compose1.yml -f compose2.yml -f compose3.yml` etc.. 

This could get wrapped up in a Makefile as well so that things are still "simple single command"